SignInCheckIn/SignInCheckIn/Services/RoomService.cs

```diff
@@ -35,5 +45,68 @@ public EventRoomDto CreateOrUpdateEventRoom(string authenticationToken, EventRoo
 
             return Mapper.Map<EventRoomDto>(response);
         }
+
+        public List<AgeGradeDto> GetEventRoomAgesAndGrades(string authenticationToken, int eventId, int roomId)
```

Comment:
Lol.  Actually I knew you did the backend because you both talked about some of this on Friday. I do debate on saying stuff like this because the code is good. Just like my statement on making a nav piece its own component.